packages/serverpod/lib/src/server/log_manager/log_writers.dart

```diff
@@ -229,3 +229,45 @@ class StdOutLogWriter extends LogWriter {
     return _logId;
   }
 }
+
+@internal
+class MultipleLogWriter extends LogWriter {
+  final List<LogWriter> _logWriters;
+
+  MultipleLogWriter(this._logWriters);
+
+  @override
+  Future<int> closeLog(SessionLogEntry entry) async {
```

Comment:
Question: What are we using the log id for? Is it a problem that we only return the id of the first log?

Reply:
@hampuslavin Since we want to minimize changes, @Isakdl and I agreed to keep it this way. In the current implementation, the database log writer has priority because, when persistent logging is enabled, the database writer is always added first to the list of log writers. This ensures that the database log ID is prioritized. You can see this behavior in the following code snippet:

https://github.com/serverpod/serverpod/blob/c65546bbf0627b06effd0cbd0db2e1227f388427/packages/serverpod/lib/src/server/session.dart#L177-L189

Reply:
Ah I see, I wonder if it would make sense to do a check for `is DatabaseLogWriter` and chose the log id based on that. Then it will be agnostic to the order the loggers are added and would be more robust. It would also show the intent clearer in the `closeLog` method. We could even add a comment in the `closeLog` method for why the `DatabaseLogWriter` id is chosen.